packages/core/contracts/optimistic-asserter/implementation/OptimisticAsserter.sol

```diff
@@ -82,6 +102,24 @@ contract OptimisticAsserter is OptimisticAsserterInterface, Lockable, Ownable, M
             );
     }
 
+    /**
+     * @notice Asserts a truth about the world, using a fully custom configuration.
+     * @dev The caller must approve this contract to spend at least bond amount of currency.
+     * @param claim the truth claim being asserted. This is an assertion about the world, and is verified by disputers.
+     * @param asserter receives bonds back at settlement. This could be msg.sender (if sent from an EOA) or a contract
+     * address or any other account that the caller wants to receive the bond at settlement time.
```

Comment:
```suggestion
     * @param asserter receives bonds back at settlement. This could be msg.sender or
     * any other account that the caller wants to receive the bond at settlement time.
```